app/services/data_migrations/define_bq_apply_request_events_schema.rb

```diff
@@ -0,0 +1,19 @@
+module DataMigrations
+  class DefineBqApplyRequestEventsSchema
+    TIMESTAMP = 20210427185904
+    MANUAL_RUN = false
+
+    def change
```

Comment:
This created a table in the US data center, can that be sorted here?

And anyway, do we want to create the table from the app? If we were manipulating the data further with the app then maybe, but I'm not sure about the responsibility for creating the table belonging to the app when we're likely to have a fair bit of machinery built in BQ that relies on this data ... the app is just something to stream data into it.

🤔

Reply:
It looks like when we create the dataset we should specify the location https://github.com/googleapis/google-cloud-ruby/blob/HEAD/google-cloud-bigquery/samples/snippets/create_dataset.rb#L21
I may have rails-consoled this test US dataset into life, can we blow it away easily?

I'm not sold on doing application migrations to create this dataset and table either, happy to let that be an admin task outside the application.

Reply:
Resolving this as we've removed the table creating data migration from this PR - will be done in Cloud Studio.